tests/common/snappi/qos_fixtures.py

```diff
@@ -25,7 +25,7 @@ def prio_dscp_map(duthosts, rand_one_dut_hostname):
         Example: {0: [0], 1: [1], 2: [2], 3: [3], 4: [4] ....}
     """
     duthost = duthosts[rand_one_dut_hostname]
-    config_facts = duthost.config_facts(host=duthost.hostname,
+    config_facts = duthost.config_facts(host=duthost.hostname, asic_index=0,
```

Comment:
Why is the asic namespace hardcoded to 0?

Reply:
this was the fixture written for multi-dut infra , to take in first hostname